dbsnp-importer/src/test/resources/application.properties

```diff
@@ -15,11 +15,11 @@ spring.datasource.password=
 
 
 processor=SubSnpCoreFieldsToEvaSubmittedVariantProcessor
-dbsnpBuild=150
-batchId=11825
+dbsnp-build=150
+batch-id=11825
 assembly=Gallus_gallus-5.0
-assemblyTypes=Primary_Assembly
-pageSize=100
-variantsCollection=variants
-chunkSize=100
+assembly-types=Primary_Assembly
+page-size=100
```

Comment:
No longer needed?

Reply:
needed for the sample reader. I didn't update that because I was in the direction of getting the main step as we decide it's good, and then follow that structure in the other steps and readers, possibly parallelizing across developers.